lib/util/concerns/hud_lists_2026.rb

```diff
@@ -716,6 +716,10 @@ def funding_sources
         54 => 'HUD: Unsheltered Special NOFO',
         55 => 'HUD: Rural Special NOFO',
         56 => 'HUD: CoC Builds',
+        12 => 'HUD: Rural Housing Stability Assistance Program [Deprecated]',
```

Comment:
Tagging @eanders on changes to `lib/util` files

Reply:
Are these changes from the most-recent files, or just something we missed?
Seems reasonable to keep them around for a bit unless we have migration instructions.